python/paddle/fluid/dygraph/dygraph_to_static/ast_utils.py

```diff
@@ -360,7 +360,9 @@ def ast_to_func(ast_root, func_name, delete_on_exit=True):
     # TODO(Aurelius84): more elegant way to transform ast into callable object
     import_str = "import paddle\n" \
                  "import paddle.fluid as fluid\n" \
-                 "import paddle.fluid.layers as layers\n"
+                 "import paddle.fluid.layers as layers\n" \
+                 "import numpy as np\n" \
+                 "import numpy\n"
```

Comment:
Here may lead to conflicts with [PR 22830](https://github.com/PaddlePaddle/Paddle/pull/22830/files)